README.md

```
@@ -50,17 +58,34 @@ A proxy server that enables Cursor IDE to use GitHub Copilot's API instead of Cu
+
 ## 🔌 Configuration with Cursor IDE
 
 1. Open Cursor IDE
 2. Go to Settings > API Keys
 3. In the "Override OpenAI Base URL" section, enter:
    ```
-   http://localhost:3000
+   http://localhost:18790/v1
    ```
-4. Go to http://localhost:3000 in your browser
+4. Go to http://localhost:18790/auth.html in your browser
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)